src/components/Base/DialogBase.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { ReactNode, forwardRef } from 'react'
+import styled, { css } from 'styled-components'
+
+type Props = {
+  children: ReactNode
+  radius?: 's' | 'm'
+  className?: string
+}
+
+export const DialogBase = forwardRef<HTMLDivElement, Props>(({ radius = 'm', ...props }, ref) => {
```

Comment:
[ask] Do you need to make this component? Since you are overwriting the style when using it in `InformationPanel`, you should use Base. What do you think?

Reply:
I think so 😭
I will fix soon.